src/js/TreeView.js

```diff
@@ -7,6 +7,9 @@ function TreeViewItem(label) {
 
 	this.element = document.createElement('div');
 
+	// Store reference to itself. Needed for delegated mouseover
+	this.element.item = this;
```

Comment:
I'm not super comfortable with the idea of adding a non-standard property to a host object (in this case, a DOM element). `ng-inspector` *should* have the data necessary to get around that, it may just require some additional refactoring. @rev087 is more familiar with this section of the codebase than I am, so he may have some suggestions.